packages/fast-component-explorer/app/explorer.tsx

```diff
@@ -267,7 +277,10 @@ class Explorer extends Foundation<ExplorerHandledProps, {}, ExplorerState> {
             selectedScenarioIndex: 1,
             locationPathname,
             theme: ThemeName.light,
-            viewConfig: DesignSystemDefaults,
+            viewConfig: {
+                ...DesignSystemDefaults,
+                transparentBackground: false,
```

Comment:
This is going to become problematic if we are squashing all configs in with design system defaults, can you separate them so that this is more like:

```
viewConfig: {
    designSystem: DesignSystemDefaults,
    transparentBackground: false
}
```

Reply:
viewConfig extends design system are you proposing that viewConfig has a deisgnSystem prop?